x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/UserToken.java

```diff
@@ -86,7 +86,7 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     /**
-     * Get the authentication
+     * Get the authentication (will not be null)
```

Comment:
Why comment specifically for the `authentication` field? Other than `metadata`, all other fields cannot be `null`.

Reply:
Because I had to check that it couldn't be null when I composed the log message, and so I updated the javadoc to surface that info.